api/resources/translations/messages-fr.properties

```
@@ -1264,7 +1264,10 @@ training_cards.confirm.title = Quitter l'entraînement?
 training_cards.error.loading = Erreur lors du chargement de la formation. Veuillez contacter votre superviseur.
 training_cards.error.save = Erreur lors de l'enregistrement de la formation.
 training_cards.form.saved = Formation terminée.
-training_cards.modal.title = Changements importants
+training_materials.page.no_more_trainings = Plus de formations
```

```suggestion
training_materials.page.no_more_trainings = Plus aucune formation
```

@m5r does this suggestion sound OK for you?

---

This is correct and removes the potential confusion between "more trainings" and "no more trainings" from the "plus" but it sounds a little bit informal. I would have said `Aucune formation restante`.
@latin-panda this literally translates to `No training left`

---

Aucune formation restante
Okay!